app/forms/api/v1/appointment.rb

```diff
@@ -32,7 +32,7 @@ def create
       private
 
       def end_at
-        Time.zone.parse(start_at) + 1.hour
+        Time.zone.parse(start_at) + 70.minutes
```

Comment:
Should this `70.minutes` be extractorado somewhere elseio? 

I guess we could have one for how long we tell the users appointments last for (45), and how long they're actually allocated for in the system (70).